ginkgo/ginkgo-install.sh

```diff
@@ -1,7 +1,7 @@
 # This script installs GINKGO
-VERSION=3.12.4
+VERSION=1.8.0
 
-wget -c https://github.com/ginkgo-project/ginkgo/archive/refs/heads/public_common_kernels.zip
-unzip public_common_kernels.zip && mv ginkgo-public_common_kernels ginkgo
+wget -c https://github.com/ginkgo-project/ginkgo/archive/refs/tags/v$VERSION.zip
+unzip v$VERSION.zip && mv ginkgo-$VERSION ginkgo
 cd ginkgo && mkdir build && cd build
-cmake .. && make
+cmake -DBUILD_TESTING="OFF" -DGINKGO_BUILD_EXAMPLES="OFF" -DGINKGO_BUILD_TESTS="OFF" .. && make
```

Comment:
Using more jobs should help reducing the compile times too. 

```suggestion
cmake -DBUILD_TESTING="OFF" -DGINKGO_BUILD_EXAMPLES="OFF" -DGINKGO_BUILD_TESTS="OFF" .. && make -j 4
```

Reply:
I don't think that the GitHub runners have four cores available, but ok

Reply:
Github upgraded its runners some time ago

https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories